site/source/docs/migrating_from_3.x.md

```diff
@@ -83,25 +83,169 @@ However, we did introduce some small breaking changes. We removed:
 
 ## Updating to v5
 
-Summary:
-- Contract config
-  - Removed the deployment section (now all part of blockchain config)
-  - `contracts` renamed `deploy` to match `beforeDeploy` and `afterDeploy` and for tests
-- Blockchain config
-  - A lot of new defaults, so less that you need to configure [Source](/docs/blockchain_configuration.html#Common-Parameters)
-  - A lot of renamed parameters
-    - isDev: `miningMode: 'dev'`
-    - mineWhenNeeded: `miningMode: 'auto'`
-    - ethereumClientName: `client`
-  - New `endpoint` parameter that let's you connect to an external endpoint or configure more easily which local endpoint to start [Source](/docs/blockchain_configuration.html#Parameter-descriptions)
-  - Now is the only source for accounts [Source](/docs/blockchain_accounts_configuration.html)
-- Tests
-  - Can now configure module configuration
-    - Storage, namesystem (ENS) and communication modules can now be configured 
-      - Configure them just like in the configuration files
-      - Tests default to the `test` environment
-      - Modules are turned off by default
-      - Config merges with `test` and `default` so no need to rewrite all the provider configs
-  - `deployement` section removed
-    - You can now configure the accounts and the endpoint in the `blockchain` section
-  - `contracts` renamed `deploy` and put inside the `contracts` section
+Embark's 5th version is full of improvements, especially in the configurations, all to make your life easier.
+
+In this guide, we'll go through the different breaking changes introduced in Embark 5, that will let you upgrade from Embark 4.0. If you want to also know what new features come with Embark 5, go read our relase article here (TODO add link here).
+
+### Contract configs
```

Comment:
I know this is a little annoying, but throughout the documentation we're maintaining "Smart Contract" as terminology for smart contracts as "Contract" is too ambiguous.

So maybe we change this to something like:

```
### Better Smart Contract configurations
```